src/style.css

```diff
@@ -1 +1,57 @@
-/* Styles go here */
+@import url(https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap);
+
+html {
+  font-family: Roboto, sans-serif;
+  margin: 0;
```

Comment:
remove margin from body, not html
<img width="1299" alt="Screenshot 2022-05-16 at 16 39 49" src="https://user-images.githubusercontent.com/57598825/168605738-0a092b76-933c-4aa7-a2a3-bd7ea1e56f6b.png">